cob_bms_driver/src/power_state_aggregator.py

```diff
@@ -51,10 +55,6 @@ def temperature_cb(self, msg):
         self.last_update = rospy.Time.now()
         self.temperature = msg.data
 
-    def charging_cb(self, msg):
-        self.last_update = rospy.Time.now()
-        self.charging = msg.data
```

Comment:
So, if this is a temporary workaround and the flag will later be retrieved from bms again, don't you want to keep the subscriber - simply comment `self.charging = msg.data`?